### PR TITLE
Translations Update, please integrate

### PR DESCRIPTION
**Pull request by rgeraghty**
Please integrate latest translations

### CODE REVIEW

File: packages/@react-aria/toast/intl/he-IL.json [hunk 1]
```diff
@@ -1,4 +1,4 @@
 {
   "close": "סגור",
-  "notifications": "{count, plural, one {# notification} other {# notifications}}."
+  "notifications": "{,לספור ברבים.אחד את השני {# notification} other {# notifications}}."
```
Comment (snowystinger) [inline packages/@react-aria/toast/intl/he-IL.json:3]: this seems wrong

File: packages/@react-aria/toast/intl/hr-HR.json [hunk 2]
```diff
@@ -1,4 +1,4 @@
 {
   "close": "Zatvori",
-  "notifications": "{count, plural, one {# notification} other {# notifications}}."
+  "notifications": "{broj, množina, jedna {# notification} ostale {# notifications}}."
```
Comment (snowystinger) [inline packages/@react-aria/toast/intl/hr-HR.json:3]: same

File: packages/@react-aria/toast/intl/hu-HU.json [hunk 3]
```diff
@@ -1,4 +1,4 @@
 {
   "close": "Bezárás",
-  "notifications": "{count, plural, one {# notification} other {# notifications}}."
+  "notifications": "{szám, többes szám, egy {# értesítés} egyéb {# értesítés}}."
```
Comment (snowystinger) [inline packages/@react-aria/toast/intl/hu-HU.json:3]: same

**Comment by rgeraghty** [comment]
Thanks for catching that @snowystinger I'll close this PR and follow up with vendor to correct, then resubmit

**Comment by snowystinger** [comment]
@rgeraghty just checking in on the status of these translations. We're coming up on a release and would like to get these in.

**Pull request closed without merging**